lib/src/booleans/discussion.md

```diff
@@ -0,0 +1,29 @@
+
+```
+
+// booleanWithin(Multipoint, LineString) -> at least one point not at start or end.
+// booleanWithin(Point, LineString) -> ignore Vertices.
+// booleanWithin(Point, Polygon) -> ignore Vertices.
+// booleanWithin(MultiPoint, LineString) -> at least one inner point.
+// booleanWithin(MultiPoint, Polygon) -> at least one inner point.
+// booleanWithin(LineString, LineString) -> no check if there is a inner point.
+// booleanWithin(LineString, Polygon) -> at least one point of the line or one 
+//    point between two point of the line is not on the boundary.
+// booleanWithin(Polygon, Polygon) -> no check if there is a inner point.
+
+// booleanWithin(LineString, Polygon): TurfJs seems to have an bug. When the last
+// point of the line is outside of the polygon, the function returns true.
+// turf-boolean-within -> isLineInPoly
+
+```
+
+
+
+https://geojson.io/#map=4.84/4.83/2.16
+
```

Comment:
should this be in the merged code or should this better be in an actual github issue / discussion?

Reply:
It was actually not planned to push this file. I missed to delete this file.